README.md

```diff
@@ -12,67 +12,54 @@ Applications using broadside employ a configuration file that looks something li
 ```ruby
 Broadside.configure do |config|
   config.application = 'hello_world'
-  config.docker_image = 'lumoslabs/hello_world'
+  config.docker_image = 'docker.io/lumoslabs/hello_world'
```

Comment:
re: 
> is this new? images on dockerhub shouldn't need the domain.

i just made up some shit that looks like a quay URL to make it clear what was expected here; do you have a better suggestion?  i don't really know how that stuff works